at_client/lib/src/client/at_client_impl.dart

```diff
@@ -164,36 +166,35 @@ class AtClientImpl implements AtClient {
   Future<bool> persistPrivateKey(String privateKey) async {
     var atData = AtData();
     atData.data = privateKey.toString();
-    await _localSecondary!.keyStore!.put(AT_PKAM_PRIVATE_KEY, atData);
+    await _localSecondary.keyStore!.put(AT_PKAM_PRIVATE_KEY, atData);
     return true;
   }
 
   Future<bool> persistPublicKey(String publicKey) async {
     var atData = AtData();
     atData.data = publicKey.toString();
-    await getLocalSecondary()!.keyStore!.put(AT_PKAM_PUBLIC_KEY, atData);
+    await getLocalSecondary().keyStore!.put(AT_PKAM_PUBLIC_KEY, atData);
     return true;
```

Comment:
for example won't we get a runtime error here if _localSecondary has not been initialized?